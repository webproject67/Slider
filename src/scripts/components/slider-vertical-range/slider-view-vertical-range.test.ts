import SliderModel from '../model/slider-model';
import SliderViewVerticalRange from './slider-view-vertical-range';

const model = new SliderModel();
const slider = new SliderViewVerticalRange(model);

test('spyOn mouseDown toggle vertical range', () => {
  const somethingSpy = jest.spyOn(slider, 'toggleMouseDown');
  let evt: any;
  slider.toggleMouseDown(evt);
  expect(somethingSpy).toHaveBeenCalledTimes(1)
});

it('jest snapshots', () => {
  expect(slider.element).toMatchSnapshot()
})