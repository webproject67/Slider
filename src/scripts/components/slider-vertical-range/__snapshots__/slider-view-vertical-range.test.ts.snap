// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest snapshots 1`] = `
<div
  class="slider__wrapper"
>
  
      
  <div
    class="slider__inner slider__inner--height"
    data-testid="sliderVerticalRange"
  >
    
        
    <div
      class="slider__scale slider__scale--vertical"
    >
      
          
      <div
        class="slider__bar slider__bar--vertical-range"
        style="top:0%;height:100%"
      />
      
        
    </div>
    
        
    <div
      class="slider__toggle-vertical slider__toggle-vertical--v-min"
      style="top:0%"
    />
    
        
    <div
      class="slider__toggle-vertical slider__toggle-vertical--v-max"
      style="top:100%"
    />
    
      
  </div>
  
    
</div>
`;
